server/public/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  background-color: #020d0d;
+  height: 8vh;
+  color: whitesmoke;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  position: relative;
+  font-family: 'Poppins', sans-serif;
+}
+
+p {
+  margin: auto;
+}
```

Comment:
Instead of using positioning and margin for your header, think of it in columns.